integrations/aws/aws-elasticache-monitoring/sensu-resources.yaml

```diff
@@ -0,0 +1,91 @@
+  secrets: []
+  publish: true
+  subscriptions:
+    - aws
+    - aws/elasticache
+  interval: 60
+  timeout: 10
+  round_robin: true
+  proxy_entity_name: aws-elasticache
+  output_metric_format: prometheus_text
+  output_metric_tags: []
+  output_metric_thresholds:
+    - name: aws_elasticache_cpu_utilization_maximum
+      thresholds:
+        - max: '90'
+          status: 1
+  pipelines: []
+---
+api_version: core/v2
+type: Asset
+metadata:
+  name: sensu/sensu-cloudwatch-check:0.2.0
```

Comment:
I noticed that the most recent released version of this on Bonsai is `0.3.0`. Do you want to upgrade this before merging?

Reply:
yep... let me push a commit